Stripe/PublicHeaders/STPPaymentMethodPrzelewy24.h

```diff
@@ -0,0 +1,36 @@
+//
+//  STPPaymentMethodPrzelewy24.h
+//  StripeiOS
+//
+//  Created by Vineet Shah on 4/23/20.
+//  Copyright © 2020 Stripe, Inc. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+#import "STPAPIResponseDecodable.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+A Przelewy24 Payment Method.
+
+@see https://stripe.com/docs/sources/p24
+*/
+@interface STPPaymentMethodPrzelewy24 : NSObject <STPAPIResponseDecodable>
```

Comment:
Do we know if the API ref is just incomplete, or is this intentionally empty?

Reply:
My understanding is that P24 doesn't have any addtional params besides the required email in billing details, could you help confirm? https://site-admin.stripe.com/docs/payments/p24

Reply:
Ah maybe @mshafrir-stripe or @stulentsev-stripe has context:

It's weird that this hash is [empty](https://site-admin.stripe.com/docs/api/payment_methods/object#payment_method_object-p24) in the [API ref](https://site-admin.stripe.com/docs/api/payment_methods/create#create_payment_method-p24) and the corresponding SDK objects are empty. Why not delete them?

Reply:
In the Android SDK we didn't create an object to represent P24 params.

Reply:
Documentation is auto-generated. Guess, it doesn't have a notion of omitting empty sub-hashes. And removing them would break symmetry with `type` field. We'd then have to explain why `type` accepts value `p24`, but there's no subhash for it. Was it oversight on our part? Etc.

Reply:
Are these the first payment methods we've implemented that don't have any params?

FWIW this PR follows what @csabol-stripe did with giropay in #1553 -- perhaps he has some thoughts?

Reply:
I think we should have these (empty classes)
1) For parity with the actual API response which contains `<payment_method>: {}`
2) For parity with the docs
3) Parsing the empty hash is a reasonable broken window test
4) Easier to iterate on/add to in the future

Reply:
That sounds reasonable. FYI this is what we're doing in the Android SDK - https://github.com/stripe/stripe-android/blob/e4f8ac699422a18f3aba8d471363a6980710c039/stripe/src/main/java/com/stripe/android/model/PaymentMethodCreateParams.kt#L488